eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -471,3 +471,8 @@ jobs:
       inputs:
         artifactName: docs.ms
         path: $(Build.ArtifactStagingDirectory)/docs.ms
+
+    - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
```

Comment:
```suggestion
    - task: ComponentGovernanceComponentDetection@0
```

Reply:
While this might work right now there are sometimes conflicts with different versions of these tasks so I'd suggest keeping the full name.